iDiff/differs/differs.go

```diff
@@ -1,16 +1,65 @@
 package differs
 
-import "errors"
+import (
+	"errors"
+	"os"
+
+	"github.com/runtimes-common/iDiff/utils"
+)
 
 var diffs = map[string]func(string, string) (string, error){
 	"hist": History,
 	"dir":  Package,
+	"apt":  AptDiff,
 }
 
 func Diff(arg1, arg2, differ string) (string, error) {
 	if f, exists := diffs[differ]; exists {
-		return f(arg1, arg2)
+		if differ == "hist" {
```

Comment:
I feel like the specific differs that need this ImageToDir kind of work should be the ones to call a helper function to do this and the clean up after instead of defaulting to have all of them do this except for the few cases specified before, like "hist" (also because if there end up being more of these, we're defeating the purpose of transitioning to a mapping scheme to avoid having so many switch cases)? but note: also have it on my TODO list to make more use of reflection to invoke differs so they don't have to prescribe to the specific formula we have here so perhaps that will help with that once we get that going